homeassistant/components/renault/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""Support for Renault devices."""
+import voluptuous as vol
+
+from .const import CONF_KAMEREON_ACCOUNT_ID, DOMAIN, SUPPORTED_PLATFORMS
+from .pyzeproxy import PyzeProxy
+from .services import async_setup_services, async_unload_services
+
+CONFIG_SCHEMA = vol.Schema(
+    {DOMAIN: vol.Schema({}, extra=vol.ALLOW_EXTRA)}, extra=vol.ALLOW_EXTRA
+)
+
+
+async def async_setup(hass, config):
+    """Old way of setting up integrations."""
+    return True
+
+
+async def async_setup_entry(hass, config_entry):
+    """Load a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    pyzeproxy = PyzeProxy(hass)
+    if not await pyzeproxy.setup(config_entry.data, True):
+        return False
+
+    if config_entry.unique_id is None:
+        hass.config_entries.async_update_entry(
+            config_entry, unique_id=config_entry.data[CONF_KAMEREON_ACCOUNT_ID]
+        )
+    hass.data[DOMAIN][config_entry.unique_id] = pyzeproxy
+
+    for component in SUPPORTED_PLATFORMS:
+        hass.async_create_task(
+            hass.config_entries.async_forward_entry_setup(config_entry, component)
+        )
+
+    await async_setup_services(hass)
+
+    return True
+
+
+async def async_unload_entry(hass, config_entry):
+    """Unload a config entry."""
+    unload_ok = True
+
+    for component in SUPPORTED_PLATFORMS:
```

Comment:
There are multiple platforms added to this integration. When creating new integration, please limit the PR to just 1 platform (e.g, only a binary_sensor, climate or sensor). The others can be added in followup PRs.

This keeps the PR small and allows for a quicker/higher PR throughput.

Reply:
Removed extra platforms in 62d6b5e